wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/includes/searchpromotions_formset.js

```diff
@@ -1,13 +1,14 @@
-$(function() {
-    var panel = InlinePanel({
-        formsetPrefix: "id_{{ formset.prefix }}",
-        emptyChildFormPrefix: "{{ formset.empty_form.prefix }}",
-        canOrder: true
-    });
+$(function () {
+  // eslint-disable-next-line no-undef
+  var panel = InlinePanel({
+    formsetPrefix: 'id_{{ formset.prefix }}',
+    emptyChildFormPrefix: '{{ formset.empty_form.prefix }}',
+    canOrder: true,
+  });
 
-    {% for form in formset.forms %}
-        panel.initChildControls('{{ formset.prefix }}-{{ forloop.counter0 }}');
-    {% endfor %}
+  // {% for form in formset.forms %}
+  panel.initChildControls('{{ formset.prefix }}-{{ forloop.counter0 }}');
+  // {% endfor %}
```

Comment:
I find this JS commented Django template forloop a bit confusing, can explain this syntax? Maybe with yet another comment? Above L9:

```
// {# This script is included into a template as an inline script, therefore the Django template syntax will be executed #}
```

Reply:
ok - this is done - hopefully the comment makes sense.